Logger/FullName.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Logger;
+public record struct FullName(string FirstName, string LastName, string? MiddleName = null)
+{
+    public string FirstName { get; } = FirstName??throw new ArgumentNullException(nameof(FirstName));
```

Comment:
```suggestion
    public string FirstName { get; } = FirstName ?? throw new ArgumentNullException(nameof(FirstName));
```
Small whitespace nit